### PR TITLE
Refactor mine List table to ant design table

### PR DESCRIPTION
**Pull request by mmumby**
Swapped out ant design Row/Columns on the mine list view for ant design Table. 

This is a very basic implementation of the table, filters/sorts can be added in the future.

### CODE REVIEW

File: frontend/src/components/dashboard/MineList.js [hunk 1]
```diff
@@ -1,83 +1,132 @@
...
+      </div>
+    ),
+  },
+  {
+    title: "Permit #",
+    dataIndex: "permit",
+    width: 150,
+    render: (text, record) => (
+      <div>
+        {text && text.map(({ permit_no, permit_guid }) => <div key={permit_guid}>{permit_no}</div>)}
```
Comment (NWCalvank) [inline frontend/src/components/dashboard/MineList.js]: Does this add a linting camelCase error? We'll need to disable that rule if we're going to write our code this way anyway

File: frontend/src/components/dashboard/MineList.js [hunk 2]
```diff
@@ -1,83 +1,132 @@
...
-  mines: {},
-  mineIds: [],
-  pageData: {},
-};
-
 class MineList extends Component {
   render() {
-    const { mines, mineIds, mineRegionHash } = this.props;
+    const data = [];
+    this.props.mineIds.forEach((id) => {
```
Comment (NWCalvank) [inline frontend/src/components/dashboard/MineList.js]: How about something like... ?
```
const data = this.props.mineIds.map((id) => ({ key: id, emptyField: String.EMPTY_FIELD, ... })
```

Comment (mmumby) [reply frontend/src/components/dashboard/MineList.js]: yess, that is much better

File: frontend/src/components/dashboard/MineList.js (hunk 1 above)
Comment (mmumby) [reply frontend/src/components/dashboard/MineList.js]: it does indeed add the linting error. 

I was thinking the same thing.

File: frontend/src/components/dashboard/MineList.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (mmumby) [reply frontend/src/components/dashboard/MineList.js]: Good catch, I agree. 

That has been updated, I aslo changed the component from stateful to stateless, as it does not currently have state and our linting rules suggested changing.

**Pull request merged**